src/libraries/Microsoft.Extensions.Configuration.EnvironmentVariables/src/EnvironmentVariablesConfigurationProvider.cs

```diff
@@ -16,6 +16,13 @@ public class EnvironmentVariablesConfigurationProvider : ConfigurationProvider
         private const string SqlAzureServerPrefix = "SQLAZURECONNSTR_";
         private const string SqlServerPrefix = "SQLCONNSTR_";
         private const string CustomConnectionStringPrefix = "CUSTOMCONNSTR_";
+        private const string PostgresqlAzureServerPrefix = "POSTGRESQLCONNSTR_";
```

Comment:
How did you pick the connection string list you are adding here? looking at https://learn.microsoft.com/en-us/azure/app-service/reference-app-settings?tabs=kudu%2Cdotnet#variable-prefixes  I am not seeing `APIHUBCONNSTR_` as example and looks there is more defined connection strings there? 

CC @maryamariyan @davidfowl

Reply:
@davidfowl what do you think about adding more prefixes? looking at your comment https://github.com/dotnet/runtime/issues/36123#issuecomment-537822041 looks to me you are not preferring adding more?

Reply:
> How did you pick the connection string list you are adding here? looking at https://learn.microsoft.com/en-us/azure/app-service/reference-app-settings?tabs=kudu%2Cdotnet#variable-prefixes I am not seeing `APIHUBCONNSTR_` as example and looks there is more defined connection strings there?
> 
> CC @maryamariyan @davidfowl

I got it from the original Issue https://github.com/dotnet/runtime/issues/36123#issue-614864809 But I think you are right I'll ref the doc you shared



> @davidfowl what do you think about adding more prefixes? looking at your comment [#36123 (comment)](https://github.com/dotnet/runtime/issues/36123#issuecomment-537822041) looks to me you are not preferring adding more?

On this the main issue here is that all major databases are currently support with this hack except PostgreSQL I think at the very least we should Add the PostgreSQL Prefix

Ideally I'd agree that an Azure App Service Specific Configuration Provider should be created for the Others

Reply:
We need a plan before we delete these. We should be able to do this on the app service side of the world now. I'll pursue that and a way to delete these from our code.

Reply:
@tarekgh  @davidfowl  please what's the way forward here are we going to allow this patch to the hack while @davidfowl  follows up on the Azure App Service Side of things or do we wait on the results of @davidfowl 's findings  to decide on how we'll proceed

I vote that we merge this update to fix the **POSTGRES ISSUE** FOR Current USERS as I can take out the other Endpoints as we wait for the Update from the App Service Team.

Reply:
@davidfowl would it make sense to allow the change in this PR till the work done in the app service side? I am not seeing this will make it any worse. It temporary can save users hours of investigations as mentioned in #36123. what you think about that? and you think we can track the work for app service side?

Reply:
@tarekgh  I see your tag, does that mean I should go ahead and strip the other Services in Prep for the POSTGRES Only Fix??

Reply:
I think we should decide about the best action that can be taken. I'll try to reply on the original issue to get consensus on such action.